src/main/java/Client.java

```diff
@@ -14,5 +23,11 @@ public static void main(String[] args) {
         .build();
 
     // Make the call to the getFlightInfo and GetStream
+    FlightDescriptor flight_desc = FlightDescriptor.command("GetFlightInfo".getBytes(StandardCharsets.UTF_8));
```

Comment:
Java code usually uses camelCase instead of snake_case

Reply:
Fixed